docs/specs/openapi.yaml

```diff
@@ -153,6 +79,314 @@ paths:
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/GetHealth"
+              examples:
+                healthy:
+                  value:
+                    type: async
+                    status-code: 200
+                    status: OK
+                    result:
+                      healthy: true
+  /v1/checks:
+    get:
+      summary: Get checks
+      tags:
+        - checks
+      description: Fetch information about specific health checks (or all of them), ordered by check name.
+      parameters:
+        - name: level
+          in: query
+          description: Filter checks by level
+          schema:
+            type: string
+            enum: [alive, ready]
+        - name: names
+          in: query
+          description: The names of the checks to get. To get multiple checks, specify this parameter multiple times. If not set, get all checks.
+          schema:
+            type: string
+      responses:
+        '200':
+          description: Get checks
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/GetChecks"
+              examples:
+                get checks:
+                  value:
+                    type: async
+                    status-code: 200
+                    status: OK
+                    result:
+                      - name: svc1-up
+                        status: up
+                        threshold: 3
+                        change-id: 91
+  /v1/changes:
+    get:
+      summary: Get changes
+      tags:
+        - changes
+      description: Fetch information for the specified changes.
+      parameters:
+        - name: select
+          in: query
+          description: Filter changes by status. Defaults to "in-progress".
+          schema:
+            type: string
+            enum: [all, in-progress, ready]
+            default: in-progress
+        - name: for
+          in: query
+          description: Filter changes for a specific service name.
+          schema:
+            type: string
+      responses:
+        '200':
+          description: Get changes
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/GetChanges'
+              examples:
+                get changes:
+                  value:
+                    type: sync
+                    status-code: 200
+                    status: OK
+                    result:
+                      - id: 42
+                        kind: perform-check
+                        summary: "Perform HTTP check \"svc1-up\""
+                        status: Doing
+                        tasks: 
+                          - id: 42
+                            kind: perform-check
+                            summary: "Perform HTTP check \"svc1-up\""
```

Comment:
Would a task inside a change normally have the same ID and summary as the change? I'm thinking it might be helpful to show a different ID and summary, to distinguish the task from the change

Reply:
Yes, this makes sense. It's possible for one change to have multiple tasks, for example, if we do `pebble start svc1 svc2`, then starting svc1 and svc2 would be in different tasks. Let me update the example here.